torch/_inductor/codegen/cpp_wrapper_cuda.py

```diff
@@ -171,6 +182,7 @@ def generate_kernel_call(
         arg_types=None,
         grid_fn: str = "grid",
         triton_meta=None,
+        grid_extra_kwargs="",
```

Comment:
grid_extra_kwargs of `generate_kernel_call` is not used for cpp wrapper, but it is used for python wrapper (wrapper.py). cpp wrapper is subclassed from WrapperCodeGen, so they need to have the same interface to avoid mypy errors on the interface mismatch.